pkg/executor/executor.go

```diff
@@ -112,6 +111,11 @@ func NewProcess(process string, args ...string) *Command {
 
 // New initializes a new Command that works with actorRunner
 func New(actorName, stdin string) *Command {
+	actorRunner, ok := os.LookupEnv("LEAPP_ACTOR_RUNNER")
```

Comment:
why to perform this for every execution? you could do this just one during the init phase - Not that this is a huge performance impact but still I don't see why having to execute this on every execution